packages/abstract-document/src/abstract-document-exporters/pdf/render.ts

```diff
@@ -586,3 +600,16 @@ function calculateTextOffset(textStyle: AD.TextStyle.TextStyle, defaultFontSize:
   const fontSize = AD.TextStyle.calculateFontSize(textStyle, defaultFontSize);
   return fontSize * position;
 }
+
+function calculateTextAlignmentOffset(alignment: AD.TextStyle.TextAlignment, compensatePdfKitSize: number): number {
```

Comment:
I suggest calculateTextAlignmentOffset is changed so it takes alignment and finalRect and returns a new AD.Rect.Rect that is compensated according to the alignment. Then the caller doesn't have to care about the details and can just use the new rect when drawing the text.  This should also help when fixing the issue above.

Reply:
I think this would create an issue where the .underline in drawHyperLink will have those changes applied too, which I think would change how long the underline is. 

I forgot to apply the alignmentOffset on the .underline though, so I will fix that